### PR TITLE
feature(Networking) - Add basic formdata handling

### PR DESCRIPTION
**Pull request by mpodwysocki**
Adds basic form data string handling

### CODE REVIEW

File: ReactWindows/ReactNative/Modules/Network/NetworkingModule.cs [hunk 1]
```diff
@@ -121,8 +121,28 @@ private RCTDeviceEventEmitter EventEmitter
...
+
+                        var stringContent = content.Value<string>("string");
+                        if (stringContent != null)
+                        {
+                            formDataContent.Add(new HttpStringContent(stringContent), fieldName);
+                        }
+                    }
+
+                    request.Content = formDataContent;
+                    // TODO: Check for gzipped body content
```
Comment (rozele) [inline ReactWindows/ReactNative/Modules/Network/NetworkingModule.cs]: Any chance we can throw if we detect gzip here?

File: ReactWindows/ReactNative/Modules/Network/NetworkingModule.cs [hunk 2]
```diff
@@ -121,8 +121,28 @@ private RCTDeviceEventEmitter EventEmitter
...
+                    // TODO: Issue #383
```
Comment (rozele) [inline ReactWindows/ReactNative/Modules/Network/NetworkingModule.cs]: 383 or 388

File: ReactWindows/ReactNative/Modules/Network/NetworkingModule.cs [hunk 3]
```diff
@@ -121,8 +121,28 @@ private RCTDeviceEventEmitter EventEmitter
...
+                    {
+                        headerData.ContentType = "multipart/form-data";
+                    }
+
+                    var formDataContent = new HttpMultipartFormDataContent();
+                    foreach (var content in formData)
+                    {
+                        var fieldName = content.Value<string>("fieldName");
+
+                        var formDataHeaders = content.Value<JArray>("headers");
```
Comment (rozele) [inline ReactWindows/ReactNative/Modules/Network/NetworkingModule.cs]: What are you doing with the `formDataHeaders`?

Comment (mpodwysocki) [inline ReactWindows/ReactNative/Modules/Network/NetworkingModule.cs]: `fieldname` is added as part of the multipart form body `HttpMultipartFormDataContent`

File: ReactWindows/ReactNative/Modules/Network/NetworkingModule.cs (hunk 2 above)
Comment (mpodwysocki) [inline ReactWindows/ReactNative/Modules/Network/NetworkingModule.cs]: #388 thanks for the correction

**Comment by rozele** [comment]
LGTM

**Pull request merged**